TrueSDK/External/TCTrueSDK.h

```diff
@@ -57,6 +57,7 @@
 
 @property (nonatomic, weak, nullable) id<TCTrueSDKDelegate> delegate;
 @property (nonatomic) TitleType titleType;
+@property (nonatomic, weak, nullable) NSString* locale;
```

Comment:
I think it makes no sense to have it weak.

Reply:
Yup. Fixed